src/corppa/utils/filter.py

```diff
@@ -40,17 +40,27 @@ def filter_pages(
     exclude_filter: dict | None = None,
 ) -> Iterator[dict]:
     """Takes a filename for a PPA full-text corpus in a format orjsonl supports
-    and a list of source ids. Returns a generator of filtered pages from the
-    full corpus corresponding to the list of ids.  Displays progress
-    with :mod:`tqdm` progress bar unless disabled.
+    and one or more options for filtering that corpus. Returns a generator of
+    filtered pages from the full corpus corresponding to the list of ids.
+    At least one filtering option must be specified.
+    Displays progress with :mod:`tqdm` progress bar unless disabled.
 
     :param input_filename: str, filename for corpus input
     :param source_ids: list of str, source ids to include in filtered pages
```

Comment:
Is the parameter`source_ids` required or optional? If it's optional, add this to the parameter description.